feature/map/src/main/java/io/github/droidkaigi/confsched2022/feature/map/Map.kt

```diff
@@ -75,38 +55,11 @@ fun Map(
                 .padding(innerPadding),
             contentAlignment = Alignment.Center
         ) {
-            AsyncImage(
-                model = MapImageUrl,
-                modifier = Modifier
-                    .padding(16.dp)
-                    .fillMaxSize(),
-                contentScale = ContentScale.Fit,
-                alignment = Alignment.Center,
+            Image(
+                painter = painterResource(id = R.drawable.map),
                 contentDescription = "Floor map",
+                modifier = Modifier.padding(16.dp)
```

Comment:
The padding is a bit different from Figma's design.
Figma: 14.25
<img width="489" alt="image" src="https://user-images.githubusercontent.com/5885032/191912114-a875d03a-e8e2-4e33-bbeb-90d09f76c0b8.png">